### ISSUE TITLE
Tablet styles, image isnt responsive

### ISSUE THREAD

**Issue by hipk0003** [hipk0003/mtm6201-midterm-alternate-master-files#2]
the images in the about page are not responsive, they getting skinnier as you size the viewport down.

**Comment by hipk0003** [comment]
@abhaysharma

**Comment by abhaysharma** [comment]
@hipk0003 I am assuming you are referencing the background images.
You will need to add background-position: center; for both about and service headers to see the center of the image. 

Line 242 and 274 of your CSS.

https://github.com/hipk0003/mtm6201-midterm-alternate-master-files/blob/8c7fdaa4874ef11ccf9ef46ebf5194fdb471fb94/css/style.css#L242

https://github.com/hipk0003/mtm6201-midterm-alternate-master-files/blob/8c7fdaa4874ef11ccf9ef46ebf5194fdb471fb94/css/style.css#L274

**Comment by hipk0003** [comment]
its for fitness-img .bowling-img .break-img and .climbing-img @abhaysharma

**Comment by abhaysharma** [comment]
@hipk0003 those images are not showing up on GitHub due to the path it should either start with a ./images or just images

https://github.com/hipk0003/mtm6201-midterm-alternate-master-files/blob/8c7fdaa4874ef11ccf9ef46ebf5194fdb471fb94/about.html#L91

**Comment by hipk0003** [comment]
@abhaysharma the images show up on my live server, i have pushed everything a while ago, no idea why its taking so long to load

**Comment by abhaysharma** [comment]
@hipk0003 your live server will work if you have /images/ as the path however on GitHub the path needs to be either ./images/ or images/ due to the server configuration.

**Comment by hipk0003** [comment]
i have made the change, can you have a look now? @abhaysharma

**Comment by abhaysharma** [comment]
@hipk0003 yes the images are working, the only issue I see is the images are being stretched when resizing the browser. That is due to the image being a flex item and by default, flex-items stretch on the cross-axis. To fix this you will need to add align-self: flex-start; to the declarations starting at line 519 in your CSS. You may close the issues once you are done.

**Comment by hipk0003** [comment]
i added  align self: flex start and there is no change, the imgs still get thin as the viewport shrinks @abhaysharma

**Comment by hipk0003** [comment]
it actually works but on the mockup the images are more horizontal so it doesnt match up @abhaysharma
